core/domain/exp_services.py

```diff
@@ -2385,8 +2442,10 @@ def get_exp_with_draft_applied(exp_id, user_id):
     updated_exploration = None
 
     if (exp_user_data and exp_user_data.draft_change_list and
-            is_version_of_draft_valid(exp_id, draft_change_list_exp_version)):
-        updated_exploration = apply_change_list(exp_id, draft_change_list)
+            are_changes_mergeable(
+                exp_id, draft_change_list_exp_version, draft_change_list)):
```

Comment:
No, because earlier we had is_version_of_draft_valid here which use to return True or False. And this are_changes_mergeable function also do the same so we don't expect an error here.